src/Microsoft.Azure.SignalR.Common/REST/JwtTokenCredentials.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Rest;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class JwtTokenCredentials : ServiceClientCredentials
+    {
+        private RestApiAccessTokenGenerator RestApiAccessTokenGenerator { get; }
+
+        public JwtTokenCredentials(string accessKey)
+        {
+            RestApiAccessTokenGenerator = new RestApiAccessTokenGenerator(new AccessKey(accessKey));
+        }
+
+
```

Comment:
extra line

Reply:
fix in https://github.com/Azure/azure-signalr/pull/992/commits/1ee9abc780816f67280c57052714e2e8087fa34e